src/System.Runtime.Serialization.Formatters/tests/BinaryFormatterTests.cs

```diff
@@ -562,7 +564,16 @@ private static void SanityCheckBlob(object obj, TypeSerializableValue[] blobs)
         {
             // These types are unstable during serialization and produce different blobs.
             if (obj is WeakReference<Point> ||
-                obj is Collections.Specialized.HybridDictionary)
+                obj is Collections.Specialized.HybridDictionary ||
+                obj is Color)
+            {
+                return;
+            }
+
+            // The blobs aren't identical because of different implementations on Unix vs. Windows.
+            if (obj is Bitmap ||
```

Comment:
But the tests do verify Windows<->Windows (and Unix<->Unix)?

Reply:
This was added by @ViktorHofer -- I guess we could add Unix specific blobs and Windows specific blobs into the test data and test them only in their corresponding platforms?

Reply:
I don't know, but I do know if we don't test it we can't rely on it working.

Reply:
We’re definitely testing serializing and deserializing the objects and rountripping them and checking their values. I think what this disables is the runtime generated blob vs the test data blob.

Reply:
I would prefer fixing the platform differences instead of adding different blobs. I know that's not easily doable because of the different implementations but it worries me a bit that the same object produces different outputs on Unix vs Windows.

The change discussed here is about disabling the blob base64 string comparison which is only an additional way of checking if serialization payloads have changed. The types serialization feature is well covered without this.

Reply:
@ViktorHofer I am afraid that may be a bit tricky. Generally gdiplus.dll / System.Drawing on Windows preserves original image data. That's not the case on libgdiplus / System.Drawing (Mono) / System.Drawing.Common. Thus the only stable blobs you can make are the ones generated on Linux (or other libgdiplus system). They should round-trip, but only until someone changes the libgdiplus algorithms :-/

Reply:
I see. That means these serialization tests are even more important as they assert the compatibility between gdiplus and libgdiplus.